core/src/main/kotlin/org/jetbrains/research/testspark/core/generation/llm/prompt/PromptBuilder.kt

```diff
@@ -114,6 +114,9 @@ internal class PromptBuilder(private var prompt: String) {
                         ClassType.INTERFACE -> "an interface implementing"
                         ClassType.ABSTRACT_CLASS -> "an abstract sub-class of"
                         ClassType.CLASS -> "a sub-class of"
+                        ClassType.DATA_CLASS -> "a sub data class of"
+                        ClassType.INLINE_VALUE_CLASS -> "a sub inline value class class of"
```

Comment:
There is a redundant "class" in line 118

Reply:
https://github.com/JetBrains-Research/TestSpark/pull/259#discussion_r1642845357